littleblog/entry/views.py

```diff
@@ -20,37 +23,17 @@ class EntryViewSet(viewsets.ModelViewSet):
     queryset = Entry.objects.all()
     permission_classes = [IsAuthenticated, IsOwnerOrReadOnly]
 
+    @action(detail=True, methods=['patch'])
```

Comment:
For "additonal API actions" better use `POST`. `PATCH` is almost always about the edit. Yes, we do edit under the hood, but for API users this action is "do smth" (POST) rather than "edit smth" (PUT/PATCH)